pkg/api/testspec.proto

```diff
@@ -0,0 +1,46 @@
+syntax = 'proto3';
```

Comment:
I don't think we should put this in our public facing api.

Reply:
I agree. Unfortunately I recall there being some issues around importing proto files from other locations in the project, which makes it difficult to put this file anywhere else...